services/wallet/transfer/commands_sequential.go

```diff
@@ -15,6 +15,53 @@ import (
 	"github.com/status-im/status-go/services/wallet/walletevent"
 )
 
+type findNewBlocksCommand struct {
+	*findBlocksCommand
+}
+
+func (c *findNewBlocksCommand) Command() async.Command {
+	return async.InfiniteCommand{
+		Interval: 13 * time.Second, // TODO - make it configurable based on chain block mining time
```

Comment:
I'm thinking that we should start linking these TODOs with tasks to be tracked, otherwise might pile up. Basically, mentioning it in a follow-up GH issue and the other way around. What do you think?